test/src/main/java/org/apache/accumulo/test/mapred/AccumuloOutputFormatIT.java

```diff
@@ -233,11 +232,9 @@ public void testMR() throws Exception {
       assertNull(e1);
 
       try (Scanner scanner = c.createScanner(table2, new Authorizations())) {
-        Iterator<Entry<Key,Value>> iter = scanner.iterator();
-        assertTrue(iter.hasNext());
-        Entry<Key,Value> entry = iter.next();
-        assertEquals(Integer.parseInt(new String(entry.getValue().get())), 100);
-        assertFalse(iter.hasNext());
+        int actual = scanner.stream().map(Entry::getValue).map(Value::get).map(String::new)
+            .map(Integer::parseInt).collect(onlyElement());
+        assertEquals(actual, 100);
```

Comment:
Expected should come first for the failure messages to be correct (saw this in several places throughout this PR).

```suggestion
        assertEquals(100, actual);
```

Reply:
Good catch, all should be addressed in cf5fb07